packages/trader/src/Stores/Modules/Trading/Helpers/proposal.ts

```diff
@@ -18,60 +45,69 @@ export const getProposalErrorField = response => {
     return el_error && isVisible(el_error) ? error_id : null;
 };
 
-export const getProposalInfo = (store, response, obj_prev_contract_basis) => {
-    const proposal = response.proposal || {};
-    const profit = proposal.payout - proposal.ask_price || 0;
-    const returns = (profit * 100) / (proposal.ask_price || 1);
-    const stake = proposal.display_value;
+export const getProposalInfo = (
+    store: TTradeStore,
+    response: PriceProposalResponse & TError,
+    obj_prev_contract_basis: TObjContractBasis
+) => {
+    const proposal = response.proposal || ({} as PriceProposalResponse['proposal']);
```

Comment:
PriceProposalResponse['proposal'] is used multiple times in this file, so it can be replaced with a shorter `Proposal` type which can be imported from api-types as well :)